tools/import_indexstores/ImportIndexstores.swift

```diff
@@ -126,11 +126,18 @@ Intermediates\.noindex/Previews/[^/]*/Intermediates\.noindex
                 )
             }
 
-            let xcodeOutputBase = xcodeExecutionRoot
+            let baseXcodeExecutionRoot = xcodeExecutionRoot
                 .split(separator: "/")
                 .dropLast(2)
                 .joined(separator: "/")
 
+            let xcodeOutputBase: String
+            if xcodeExecutionRoot.hasPrefix("/") {
+                xcodeOutputBase = "/" + baseXcodeExecutionRoot
+            } else {
+                xcodeOutputBase = baseXcodeExecutionRoot
```

Comment:
Unsure if this condition is ever possible, but added it just in case. Otherwise we can simplify this even further